build-aux/snap/snapcraft.yaml

```diff
@@ -411,7 +439,24 @@ parts:
       - snapd
     override-pull: |
       craftctl default
-      if [ -f fips-build ]; then
+      # FIPS stamp files were created in the 'snap' part build
+      if [ -f "${CRAFT_STAGE}/fips-build-lp" ]; then
```

Comment:
I have to verify that stage is always ready in the override-pull. I wonder if that can break.